app/Services/UserService.php

```diff
@@ -15,4 +16,12 @@ public function getWishlistByUserId(): array
     {
         return $this->userRepository->getWishlistByUserId();
     }
+
+    public function submitGameReview(int $game_id, int $user_id, float $game_rating, string $text_review):Reviews
+    {
+        $submittedReview = $this->userRepository->InsertGameReview($game_id, $user_id, $game_rating, $text_review);
+        $returnedReview =  new Reviews();
+        $returnedReview = $submittedReview;
```

Comment:
why not just `return $this->userRepository->InsertGameReview($game_id, $user_id, $game_rating, $text_review);`